src/main/java/robot/subsystems/drivetrain/pure_pursuit/Constants.java

```diff
@@ -24,9 +24,17 @@
     public final static double kA = 0.002;
     public final static double kP = 0.01;
 
+    // fall control
+    public final static double Kj = 0;
+    public final static double Kam = 0;
```

Comment:
What does `Kam` mean? give it a better name.
Also, If it is a constant, it should be `ALL_CAPS_WITH_UNDERSCORES`. Else, it should be `camelCase`.

Reply:
the maximum mistake in the desired acceleration ,and this is fixed